static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -71,7 +73,33 @@ define([
         }
     }
 
+
+
+    function shouldAutoplay(){
+
+        function isMobile() {
+            const isIOS = () => /(iPad|iPhone|iPod touch)/i.test(navigator.userAgent);
```

Comment:
this is a `javascripts-legacy` module - do we want to be writing ES6 here @sndrs ?